src/rest-server/src/models/v2/job/k8s.js

```diff
@@ -1142,6 +1186,63 @@ const getSshInfo = async (frameworkName) => {
   );
 };
 
+const addTag = async (frameworkName, tag) => {
+  // check if frameworkName exist
+  const framework = await databaseModel.Framework.findOne({
+    where: { name: encodeName(frameworkName) },
+  });
+
+  if (framework) {
+    // add tag
+    const data = await databaseModel.Tag.findOrCreate({
+      where: {
+        frameworkName: encodeName(frameworkName),
+        name: tag,
```

Comment:
Check the tag. it shouldn't have `,`

Reply:
> Check the tag. it shouldn't have `,`

Added a middleware here : https://github.com/microsoft/pai/pull/4853/files#diff-1e4d6d8193e8e8f5095fd5e5102c0a22